safe/impact_functions/generic/classified_polygon_people/test/test_classified_polygon_people.py

```diff
@@ -0,0 +1,99 @@
+# coding=utf-8
+"""
+InaSAFE Disaster risk assessment tool developed by AusAid and World Bank
+- *Classified Hazard Land Cover Impact Function Test Cases.**
+
+Contact : ole.moller.nielsen@gmail.com
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+
+"""
+
+import unittest
+from qgis.core import QgsVectorLayer
+
+from safe.impact_functions.generic.classified_polygon_people. \
+    impact_function import ClassifiedPolygonHazardPolygonPeopleFunction
+from safe.impact_functions.impact_function_manager import ImpactFunctionManager
+from safe.test.utilities import get_qgis_app, test_data_path
+from safe.storage.utilities import safe_to_qgis_layer
+
+QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
+
+
+class TestClassifiedPolygonPeopleFunction(unittest.TestCase):
+    """Test for Classified Polygon People Impact Function."""
+
+    def setUp(self):
+        registry = ImpactFunctionManager().registry
+        registry.clear()
+        registry.register(ClassifiedPolygonHazardPolygonPeopleFunction)
+
+    def test_run(self):
```

Comment:
Please give some comment/docstring about this test. See other IF's test for reference.